core-java-modules/core-java-os/src/test/java/com/baeldung/java9/process/ProcessAPIEnhancementsUnitTest.java

```diff
@@ -61,7 +61,7 @@ public void givenSpawnProcess_whenInvokeGetInfo_thenSuccess() throws IOException
           .contains("java"));
         assertEquals(true, processInfo.startInstant()
           .isPresent());
-        assertEquals(true, processInfo.totalCpuDuration()
+        assertEquals(false, processInfo.totalCpuDuration()
```

Comment:
have the assertions changed because the Java version has been upgraded? I just want to understand if we are asserting the correct data.

Reply:
yes